test/system/012-manifest.bats

```diff
@@ -121,7 +121,7 @@ EOF
     local manifestlocal="test:1.0"
     run_podman manifest create $manifestlocal
     for arch in amd arm;do
-        # FIXME: --layers=false needed to work around #19860
+        # This leaves behind a <none>:<none> image that must be purged, below
```

Comment:
Can somebody explain me _why_ this leaves an untagged image behind?  I was trying to figure it out but don´t understand it yet.

Reply:
Usually means there was an aaa image there before the build was run.

Reply:
I think `from scratch` creates an image. And that is the <none>
@flouthoc Thoughts?

Reply:
This is a intermediate image generated from the build since `--layers=true` now.

Reply:
The <none>:<none> is committed intermediate image for the 2nd step of the build, this is expected.

Reply:
Docker does not leave this image behind.

Reply:
docker build does not support --arch BTW.
```
#  printf "FROM scratch\nARG TARGETARCH\nCOPY Makefile /foo\n" | docker buildx build -t aaa --arch amd64 -f - .
unknown flag: --arch
See 'docker buildx build --help'.
```

Reply:
I think legacy `docker`( not buildx ) should leave an intermediate image behind, buildkit has different cache mechanism. Instead of `--arch` i think direct `--platform` can be used with docker.